client/app/lib/create-post-notification.ts

```diff
@@ -0,0 +1,40 @@
+import { prisma } from "./prisma.server";
+
+interface Options {
+	message: string;
+	actorId: number;
+	entityId: number;
+}
+
+async function createPostNotification({ message, actorId, entityId }: Options) {
+	const suscribers = await prisma.post.findMany({
+		where: {
+			OR: [{ id: entityId }, { parentId: entityId }],
+			AND: { userId: { not: actorId } },
+		},
+		distinct: ["userId"],
+		select: { userId: true },
+	});
+
+	const notification = await prisma.notification.create({
+		data: {
+			message,
+			actorId,
+			entityId,
+			entityType: "post",
+		},
+	});
+
+	await Promise.all(
```

Comment:
```suggestion
	await Promise.allSettled(
```

Because using just `.all` will cause other promises to reject when one of them fails.

If some fails, no problem. It's non-consequential, so let's use `.allSettled`

Reply:
Alright